schema/collection.go

```diff
@@ -51,9 +52,11 @@ type DefaultCollection struct {
 	Validator *jsonschema.Schema
 	// JSON schema
 	Schema jsoniter.RawMessage
-
 	// search schema
 	Search *tsApi.CollectionSchema
+	// QueryableFields are similar to fields but these are flattened form of fields i.e. for an object there may be more
```

Comment:
Nit: since you are comparing the types, this should probably say "are similar to Fields", with an upper case.

Consider: "QueryableFields are similar to Fields but they represent them in a flattened form. In other words, for an object there...